addons/event/views/event_registration_views.xml

```diff
@@ -132,44 +133,15 @@
         <field name="model">event.registration</field>
         <field name="priority">10</field>
         <field name="arch" type="xml">
-            <kanban class="o_event_attendee_kanban_view" default_order="name, create_date desc" sample="1">
+            <kanban class="o_event_attendee_kanban_view" default_order="name, create_date desc" sample="1" js_class="registration_summary_dialog_kanban">
                 <field name="name"/>
                 <field name="partner_id"/>
                 <field name="state"/>
                 <field name="email"/>
                 <field name="event_ticket_id"/>
                 <field name="active"/>
+                <field name="barcode"/>
                 <templates>
-                    <t t-name="event_attendees_kanban_icons_desktop">
```

Comment:
Would keep the content, as people may use it in custom code. Just remove it in master.